### DISCUSSION TITLE
vcenterreceiver not part of rpm package

### DISCUSSION THREAD

**Discussion by dimitrisfinas** [open-telemetry/opentelemetry-collector-contrib#12177] [Q&A] [answered]
hello,

I noticed the vcenter receiver is not yet part of the the RPM package for the Otel collector contrib, found here: https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.54.0/otelcol-contrib_0.54.0_linux_amd64.rpm

I wanted to know if you have better visibility of when it will be included in it?
And, in general, do you have a roadmap on this receiver (like when is it planned to be beta, ga, etc...)?

Thanks for your help,

**Comment by codeboten** [comment]
Thanks @dimitrisfinas, once components implementations are complete, they're usually added to the contrib distribution. I'm not familiar with the specifics of the vcenter receiver, pinging code owners @djaglowski @schmikei. Can this component be added to the contrib distribution?

**Comment by djaglowski** [comment, marked as answer]
Yes, it can be added. I'll make a PR

**Reply by WagnerCarvalho** [reply]
thank you so much

**Reply by dimitrisfinas** [reply]
Thanks @djaglowski, may I please ask you link to the PR or if you know if it will be integrated in next version (v0.55 I think) or an update to existing v0.54 ? So that I know when I can download a version to test it.

**Reply by djaglowski** [reply]
Here is the PR: [releases#161](https://github.com/open-telemetry/opentelemetry-collector-releases/pull/161)

The component will be included in the next release (v0.56.0, scheduled to be released on July 20th)

**Reply by codeboten** [reply]
@djaglowski I believe you meant v0.56.0 :)

**Reply by djaglowski** [reply]
Yes, thanks for the correction. Fixed it!

**Reply by dimitrisfinas** [reply]
hi, I downloaded v0.56 and despite the vcenter receiver seems to be in manifest, it doesn't start. Each time I add the vcenter receiver with default configuration, my collector doesn't start with no error I can see (this is on CentOS VM and using the collector rpm package):Jul 22 13:18:18 instance-1 systemd[1]: Started OpenTelemetry Collector Contrib.
Jul 22 13:18:18 instance-1 otelcol-contrib[2663]: 2022/07/22 13:18:18 proto: duplicate proto type registered: jaeger.api_v2.PostSpansRequest
Jul 22 13:18:18 instance-1 otelcol-contrib[2663]: 2022/07/22 13:18:18 proto: duplicate proto type registered: jaeger.api_v2.PostSpansResponse
Jul 22 13:18:18 instance-1 systemd[1]: otelcol-contrib.service: main process exited, code=exited, status=1/FAILURE
Jul 22 13:18:18 instance-1 systemd[1]: Unit otelcol-contrib.service entered failed state.
Jul 22 13:18:18 instance-1 systemd[1]: otelcol-contrib.service failed.
Jul 22 13:18:18 instance-1 systemd[1]: otelcol-contrib.service holdoff time over, scheduling restart.
Jul 22 13:18:18 instance-1 systemd[1]: Stopped OpenTelemetry Collector Contrib.

**Reply by djaglowski** [reply]
You may need to do some digging, but I suspect you'll find the collector logs are being written elsewhere by systemd.

Since you stated you're using the default configuration, it's worth noting that several fields are required, so this would not be expected to work as is.

**Reply by dimitrisfinas** [reply]
I switched by starting it in command line first ("/usr/bin/otelcol-contrib --config=/etc/otelcol-contrib/config.yaml") before putting it in daemon mode, so that I can debug more easily. Now, I can see the expected vcenter receiver errors directly in console and see it is linked to missing values and incorrect connections. Thx for feedback
